ghcide/test/exe/HighlightTests.hs

```diff
@@ -55,7 +55,12 @@ tests = testGroup "highlight"
           ]
         highlights <- getHighlights doc (Position 3 17)
         liftIO $ highlights @?=
-          [ DocumentHighlight (R 3 17 3 23) (Just DocumentHighlightKind_Write)
+          [
+            DocumentHighlight (R 3 11 3 14) (Just DocumentHighlightKind_Write)
+          , DocumentHighlight (R 3 17 3 23) (Just DocumentHighlightKind_Read)
+          , DocumentHighlight (R 3 32 3 38) (Just DocumentHighlightKind_Read)
+          , DocumentHighlight (R 4 4 4 7) (Just DocumentHighlightKind_Read)
+          , DocumentHighlight (R 3 17 3 23) (Just DocumentHighlightKind_Write)
```

Comment:
are these outputs correct?

Reply:
good point should we highlight the constructor if we just hover on its field?

Reply:
Is it the desire behaviour? If not I think we should kept the broken test and open an issue for it instead!

Reply:
I don't think this is desire, but a fix is relatively easy. I have impelmented it.